extensions/blocks/contact-form/components/jetpack-field.js

```diff
@@ -4,59 +4,38 @@
 import classNames from 'classnames';
 import { __ } from '@wordpress/i18n';
 import { Fragment } from '@wordpress/element';
-import { InspectorControls } from '@wordpress/block-editor';
-import { PanelBody, TextControl } from '@wordpress/components';
+import { TextControl, Disabled } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import JetpackFieldLabel from './jetpack-field-label';
+import JetpackFieldControls from './jetpack-field-controls';
+
+export default function JetpackField( props ) {
+	const { isSelected, type, required, label, setAttributes, placeholder } = props;
 
-function JetpackField( {
-	isSelected,
-	type,
-	required,
-	label,
-	setAttributes,
-	defaultValue,
-	placeholder,
-	id,
-} ) {
 	return (
 		<Fragment>
 			<div className={ classNames( 'jetpack-field', { 'is-selected': isSelected } ) }>
-				<TextControl
-					type={ type }
-					label={
-						<JetpackFieldLabel
-							required={ required }
-							label={ label }
-							setAttributes={ setAttributes }
-							isSelected={ isSelected }
-						/>
-					}
-					placeholder={ placeholder }
-					value={ placeholder }
-					onChange={ value => setAttributes( { placeholder: value } ) }
-					title={ __( 'Set the placeholder text', 'jetpack' ) }
+				<JetpackFieldLabel
+					required={ required }
+					label={ label }
+					setAttributes={ setAttributes }
+					isSelected={ isSelected }
 				/>
-			</div>
-			<InspectorControls>
-				<PanelBody title={ __( 'Field Settings', 'jetpack' ) }>
-					<TextControl
-						label={ __( 'Default Value', 'jetpack' ) }
-						value={ defaultValue }
-						onChange={ value => setAttributes( { defaultValue: value } ) }
-					/>
+				<Disabled>
 					<TextControl
-						label={ __( 'ID', 'jetpack' ) }
-						value={ id }
-						onChange={ value => setAttributes( { id: value } ) }
+						type={ type }
+						placeholder={ placeholder }
+						value={ placeholder }
+						onChange={ value => setAttributes( { placeholder: value } ) }
+						title={ __( 'Set the placeholder text', 'jetpack' ) }
 					/>
-				</PanelBody>
-			</InspectorControls>
+				</Disabled>
```

Comment:
Is there a reason why this is disabled?

Reply:
Yes, to avoid the fields being editable or being clicked on. It seems to be the easiest way to allow the user to click on a field and select the containing block.

Reply:
Right, but then it prevents changing the placeholder, doesn't it? 🤔 
If that's intended, I guess we could lose the placeholder attribute altogether.